scripts/mail-all-activity.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+'''Mail people to check their activity based on YAML dump of Amy database.'''
+
+import sys
+import yaml
+
+
+USAGE = 'Usage: mail-all-activity [-r|--real] [filename]'
+
+BODY = '''We would like to bring our records up to date, and would be grateful
+if you could check the information below and send us corrections or additions.
+
+name: {name}
+preferred email: {email}
+became instructor: {became_instructor}
+{optional}
+{skills}
+{tasks}'''
+
+
+def main(prog_name, args):
+    '''Main driver.'''
+
+    # Default is dummy run - only actually send mail if told to.
+    sender = display
+    if args[0] in ('-r', '--real'):
+        sender = send
+        args = args[1:]
```

Comment:
I'd move this `if` into the chunk of code that checks length of `args` (lines 34-36).